etl/data_helpers/geo.py

```diff
@@ -837,7 +849,10 @@ def add_gdp_to_table(
     return tb_with_gdp
 
 
-def create_table_of_regions_and_subregions(ds_regions: Dataset, subregion_type: str = "members") -> Table:
+@cache
+def create_table_of_regions_and_subregions(
```

Comment:
@pabloarosado I enabled runtime cache for this function, it was called multiple times with the same params and caching it reduced runtime of
```
etlr data://garden/animal_welfare/2025-01-23/number_of_farmed_crustaceans --force --only
```
by about a second